lib/hydra/works/models/concerns/contained_files.rb

```diff
@@ -0,0 +1,17 @@
+module Hydra::Works::ContainedFiles
+  extend ActiveSupport::Concern
+
+  included do
+    directly_contains :files, has_member_relation: ::RDF::URI("http://pcdm.org/hasFile"), class_name: "Hydra::Works::File"
+  end
+
+  def thumbnail
+    files = []
```

Comment:
files is a confusing name for a local variable since we already have a files method that we're calling on line 10? how about `matches`?